src/lib/support/Pool.cpp

```diff
@@ -127,6 +127,46 @@ Loop StaticAllocatorBitmap::ForEachActiveObjectInner(void * context, Lambda lamb
     return Loop::Finish;
 }
 
+size_t StaticAllocatorBitmap::FirstAllocatedIndex()
+{
+    size_t idx = 0;
+    for (size_t word = 0; word * kBitChunkSize < Capacity(); ++word)
+    {
+        auto & usage = mUsage[word];
+        auto value   = usage.load(std::memory_order_relaxed);
+        for (size_t offset = 0; offset < kBitChunkSize && offset + word * kBitChunkSize < Capacity(); ++offset)
+        {
+            if ((value & (kBit1 << offset)) != 0)
+            {
+                return idx;
+            }
+            idx++;
+        }
+    }
+    VerifyOrDie(idx == mCapacity);
+    return mCapacity;
+}
+
+size_t StaticAllocatorBitmap::NextActiveIndexAfter(size_t start)
```

Comment:
Ah, ok, now I see how this works.

One question: Why is this function using "ActiveIndex" while the one just above uses "AllocatedIndex", just terminology-wise?